website/src/theme/DocPage/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
This file is copied from Facebook?

Reply:
Generate when executing `docusaurus swizzle @docusaurus/theme-classic DocPage`